### PR TITLE
Update QueryNetwork tests

### PR DESCRIPTION
**Pull request by mandrono**
Also need fix tests for HDDL and ARM plug-in

product-config PR: 1758

### CODE REVIEW

**Comment by mandrono** [comment]
@iefode take a look please

File: inference-engine/tests/functional/plugin/shared/include/hetero/query_network.hpp [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace HeteroTests {
 
 using QueryNetworkTestParameters = std::tuple<
     std::string,
-    std::shared_ptr<ngraph::Function>
+    std::pair<std::set<std::string>, std::shared_ptr<ngraph::Function>>
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/include/hetero/query_network.hpp:17]: Please, describe parameters

**Comment by iefode** [comment]
CC @ilya-lavrenov

**Review by ilya-lavrenov — APPROVED** [review]

**Review by apankratovantonp — APPROVED** [review]

File: inference-engine/tests/functional/plugin/shared/include/hetero/query_network.hpp (hunk 1 above)
Comment (mandrono) [reply inference-engine/tests/functional/plugin/shared/include/hetero/query_network.hpp]: done

**Pull request closed without merging**